addons/dogtags/functions/fnc_getDogtagData.sqf

```diff
@@ -19,13 +19,24 @@ private _dogTagData = _target getVariable QGVAR(dogtagData);
 if (!isNil "_dogTagData") exitWith {_dogTagData};
 
 // Create dog tag data once for the unit: nickname, code (eg. 135-13-900) and blood type
+private ["_bloodTypeIndex", "_code"];
+//hint getplayeruid _target;
+if ((typename getplayeruid _target) == "SCALAR") then {
```

Comment:
`getPlayerUID` is the correct case. As far as I can tell it will always report a string, so it will never be `typeOf` `SCALAR`